dev/workflow/applications/gefs.py

```diff
@@ -169,6 +172,9 @@ def get_task_names(self):
         if options['nens'] > 0:
             tasks += ['atmos_ensstat']
 
+        if options['do_awips']:
+            tasks += ['awips']
+
```

Comment:
Since awips depends on ensstat, I suggest putting the `do_awips` conditional inside the `nens` conditional.  

```suggestion
        if options['nens'] > 0:
            tasks += ['atmos_ensstat']
            if options['do_awips']:
                tasks += ['awips']

```

Reply:
@EricSinsky-NOAA Done.